bundle/src/main/java/com/adobe/acs/commons/workflow/impl/WorkflowPackageManagerImpl.java

```diff
@@ -166,33 +165,36 @@ public final Page create(final ResourceResolver resourceResolver, String bucketS
 
         if (resource == null) {
             log.warn("Requesting paths for a non-existent Resource [ {} ]; returning empty results.", path);
-            return Collections.EMPTY_LIST;
+            return Collections.emptyList();
+        }
 
-        } else if (!isWorkflowPackage(resourceResolver, path)) {
+        if (!isWorkflowPackage(resourceResolver, path)) {
             log.debug("Requesting paths for a non-Resource Collection  [ {} ]; returning provided path.", path);
-            return Arrays.asList(new String[]{ path });
```

Comment:
It was just a simplification since there was no need for `if ... else` where there was a `return` anyway in each block. Retracted it for the PR's simplicity.